example/notify_inval_entry.c

```diff
@@ -78,13 +78,15 @@
 
 #define FUSE_USE_VERSION 34
 
+#include <fuse_kernel.h>
```

Comment:
What's this needed for? I think this should only be needed when communicating directly with the kernel, not when going through libfuse.

Reply:
think that was part of the `fuse_i.h` needed to be able to set `se->error` which is not needed anymore